components/developer/subversion/Makefile

```diff
@@ -151,7 +149,7 @@ COMPONENT_TEST_MASTER = \
 	$(COMPONENT_TEST_RESULTS_DIR)/results-$(MACH64).master
 
 # Build dependencies
-REQUIRED_PACKAGES += library/py3c
+PYTHON_REQUIRED_PACKAGE += library/python/py3c-39
 
```

Comment:
No, the -39 needs to be removed here because the python version will be automatically determined and added.